index.d.ts

```diff
@@ -1110,6 +1110,13 @@ declare namespace algoliasearchHelper {
      */
     sortFacetValuesBy?: 'count' | 'alpha';
 
+    /**
+     * The relevancy threshold to apply to search in a virtual index [0-100]. A Bigger
+     * value means fewer, but more relevant results, smaller value means more, but
+     * less relevant results.
```

Comment:
Is this the official documentation?

```suggestion
     * The relevancy threshold to apply to search in a virtual index [0-100]. A bigger
     * value means fewer but more relevant results, a smaller value means more but
     * less relevant results.
```

Reply:
There's no official documentation yet.